<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1548</width>
    <height>939</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OCRtoODT GUI</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutCentral">
    <property name="spacing">
     <number>6</number>
    </property>

    <item>
     <widget class="QSplitter" name="splitterMain">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>

      <!-- LEFT -->
      <widget class="QWidget" name="panelLeft">
       <layout class="QVBoxLayout" name="verticalLayoutLeft">
        <item>
         <widget class="QLabel" name="lblFiles">
          <property name="text">
           <string>File list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listFiles"/>
        </item>
       </layout>
      </widget>

      <!-- CENTER -->
      <widget class="QWidget" name="panelCenter">
       <layout class="QVBoxLayout" name="verticalLayoutCenter">
        <item>
         <widget class="QLabel" name="lblPreview">
          <property name="text">
           <string>Preview</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="viewPreview">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>400</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutZoomButtons">
          <item>
           <widget class="QPushButton" name="btnZoomIn">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnZoomOut">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnZoomFit">
            <property name="text">
             <string>Fit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnZoom100">
            <property name="text">
             <string>100%</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>

      <!-- RIGHT -->
      <widget class="QWidget" name="panelRight">
       <layout class="QVBoxLayout" name="verticalLayoutRight">
        <item>
         <widget class="QLabel" name="lblOcrResultTitle">
          <property name="text">
           <string>OCR Result</string>
          </property>
         </widget>
        </item>

        <!-- Tabs removed, only OCR text list stays -->
        <item>
         <widget class="QListView" name="listOcrText"/>
        </item>
       </layout>
      </widget>

     </widget>
    </item>
   </layout>
  </widget>

  <!-- STATUS BAR -->
  <widget class="QStatusBar" name="statusBar">
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ready</string>
    </property>
   </widget>

   <widget class="QProgressBar" name="progressTotal">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>

   <widget class="QLabel" name="lblOcrState">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(220, 80, 80); font-weight: bold;</string>
    </property>
    <property name="text">
     <string>OCR in progressâ€¦</string>
    </property>
   </widget>
  </widget>

  <!-- TOOLBAR (unchanged) -->
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionClear"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
  </widget>

  <!-- ACTIONS with icons restored -->
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/open.svg</normaloff>:/icons/icons/open.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>

  <action name="actionClear">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/clear.svg</normaloff>:/icons/icons/clear.svg</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>

  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/settings.svg</normaloff>:/icons/icons/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>

  <action name="actionRun">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/run.svg</normaloff>:/icons/icons/run.svg</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>

  <action name="actionStop">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/stop.svg</normaloff>:/icons/icons/stop.svg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>

  <action name="actionExport">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/export.svg</normaloff>:/icons/icons/export.svg</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>

  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/about.svg</normaloff>:/icons/icons/about.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>

  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/icons/help.svg</normaloff>:/icons/icons/help.svg</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>

 </widget>

 <!-- IMPORTANT: resources include restored -->
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>

 <connections/>
</ui>

